### ISSUE TITLE
Docker fails on boot with 'Fatal Error vmstateevent invalid json:'

### ISSUE THREAD

**Issue by mattrayner** [docker/for-mac#272]
### Expected behavior
Docker should begin as expected

### Actual behavior
Docker crashes with fatal error 'vmstateevent invalid json:'

### Information
Occasionally if I reset docker to factory defaults it will work once. Ususally only once though

Diagnostic ID: 44EDAAE0-A493-44EA-9C0B-BDD832F76AE9
Docker for Mac: 1.12.0-a (Build 11213)
macOS: Version 10.12 (Build 16A286a)
[ERROR]  docker-cli
         docker-cli check failed with: Failure("docker -v: timeout after 10.00s")
[OK]     app
[OK]     moby-syslog
[OK]     disk
[OK]     virtualization
[OK]     system
[OK]     menubar
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     db
[ERROR]  slirp
         Connection refused (ECONNREFUSED) connecting to /Users/mattrayner/Library/Containers/com.docker.docker/Data/s51: check if service is running
         com.docker.slirp is not running
[OK]     moby-console
[OK]     logs
[OK]     vmnetd
[OK]     env
[OK]     moby
[ERROR]  driver.amd64-linux
         Docker.app/Contents/MacOS/com.docker.driver.amd64-linux -db is not running

### Steps to reproduce

1. Install Docker for Mac
2. Allow it to auto-start on boot

**Comment by dsheets** [comment]
Duplicate of #64.

**Comment by djs55** [comment]
Docker for Mac beta 26 has improved support for macOS 10.12 Sierra: all the major show stopper issues which we are aware of have been fixed.

If you have the opportunity, please give it a try on the latest Sierra GM seed and open fresh issues for any problems you find.

- Release notes: https://download.docker.com/mac/beta/1.12.1.12048/NOTES
- Download: https://download.docker.com/mac/beta/Docker.dmg

Thanks for your help testing Docker for Mac with Sierra!
